connman/plugins/vpn.c

```diff
@@ -754,6 +767,9 @@ static void add_connection(const char *path, DBusMessageIter *properties,
 			/* Ignored */
 		} else if (g_str_equal(key, "UserRoutes")) {
 			/* Ignored */
+		} else if (g_str_equal(key, "AuthErrorLimit")) {
+			dbus_message_iter_get_basic(&value,
+						&data->auth_error_limit);
```

Comment:
Yeah, might be good to do an overhaul of this because there are places where ints do get assigned directly from D-Bus messages. Changed this in 0fe52aa9 to have simple typed getter. Maybe later make it more complex.